spec/support/schema.graphql

```diff
@@ -0,0 +1,95 @@
+type Mutation {
+  createOrganization(data: NewOrganization!): Organization
+  deleteAccount(password: String!): Boolean
+  deleteOrganization(id: ID!): Boolean
+  saveAccount(data: UserChangeset!, password: String!): User
+  saveOrganization(id: ID!, data: OrganizationChangeset!): Organization
+}
```

Comment:
I don't have the picture of the whiteboard with our notes of last week. Are these the types we agreed upon? They look reasonable.